lib/ilp.js

```diff
@@ -96,10 +102,15 @@ class Ilp {
 
     this.log.info('Connection created successfully')
 
-    await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+    let { success, remainingAmount } = await this.sendMoney({ ilpAmount: ilpScaledAmount, connection })
+    // Reset remaining amount to millicents, where as ILP operates in nanodollars
+    if (remainingAmount > 0) remainingAmount = remainingAmount / 10000
 
     this.log.info('Closing connection to ILP connector...')
     await connection.end()
+
+    // Pass along success and remaining amount to caller so they can handle it
+    return { success: !!success, remainingAmount: remainingAmount || 0 }
```

Comment:
nit: the `!!` seems redundant since success is always a bool (and we're the only ones who ever set it)